OWNERS

```diff
@@ -1,26 +1,25 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-- aojea # lead
-- BenTheElder # lead
+- aojea
+- BenTheElder
 - dims
-- stevekuznetsov
-- cjwagner # oncall, lead
-- mpherman2 # oncall
-- krzyzacy
+- upodroid
 approvers:
-- aojea # lead
-- BenTheElder # lead
+- aojea
+- BenTheElder
 - dims
-- stevekuznetsov
-- krzyzacy
-- test-infra-oncall # oncall
 - upodroid
 emeritus_approvers:
 - amwat
+- cjwagner
```

Comment:
this and some of the others are from the oncall alias, these folks did review and approve a lot of changes over the years, but were not necessarily listed inline here versus the alias

the alias is removed entirely as the concept is gone, so documenting here.